src/pages/workspaces/workspace/analysis/modals/ComputeModal/persistent-disk-controls.ts

```diff
@@ -0,0 +1,8 @@
+import { Ajax } from 'src/libs/ajax';
+import Events from 'src/libs/events';
+
+// TODO: Where should this live? Should this be in the modal container?
+export const handleLearnMoreAboutPersistentDisk = ({ setViewMode }) => {
+  setViewMode('aboutPersistentDisk');
+  Ajax().Metrics.captureEvent(Events.aboutPersistentDiskView);
```

Comment:
Possibly adjust the setViewMode passthrough to be onClick: handleLearnMoreAboutPersistentDiskSection.

Reply:
to follow up on the setViewMode comment I made elsewhere, removing it from other places looks better... but I have two thoughts here:

1) We might want to track a different captureEvent event code for GCP "learnMore" versus Azure?  Or do they show the same content?

2) This is pretty close to a 1-liner, which is often not worth DRY concerns... and setViewMode happening here still seems a bit disconnected... at a minumum, 'aboutPersistentDisk' string value should be a constant defined somewhere since I assume the two modals set/read that same value and we want them to stay in sync.... and well, that  just makes it feel even more like the respective modals should just handle this small bit of logic themselves....

Reply:
1. The resulting info is the same, but I don't see why we wouldn't want to capture the cloud platform. 
2. I'll move the function then. It'll look like:

```
onClick: () => {
  setViewMode('aboutPersistentDisk');
  Ajax().Metrics.captureEvent(Events.aboutPersistentDiskView, { cloudPlatform: cloudProviderTypes.GCP });
}
```

in the modals (With GCP -> Azure for azure.)

Reply:
Unsure where the 'aboutPersistentDisk' should live. Probably under ComputeModal/ComputeModalConstants.ts ?